src/govuk/components/tabs/_index.scss

```diff
@@ -30,16 +30,11 @@
   }
 
   .govuk-tabs__tab {
+    @include govuk-link-common;
     @include govuk-link-style-default;
 
     display: inline-block;
     margin-bottom: govuk-spacing(2);
-
-    // Focus state for mobile and when JavaScript is disabled
-    // It is removed for JS-enabled desktop styles
-    &:focus {
-      @include govuk-focused-text;
-    }
```

Comment:
Could this be done in a separate PR, before the changes to the link styles? It seems to me that it could be, but I might be misunderstanding.